Kitodo-DataAccess/src/main/java/org/kitodo/dataaccess/format/xml/XMLWriter.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess.format.xml;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.TreeMap;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import org.kitodo.dataaccess.AccessibleObject;
+import org.kitodo.dataaccess.IdentifiableNode;
+import org.kitodo.dataaccess.LangString;
+import org.kitodo.dataaccess.Literal;
+import org.kitodo.dataaccess.Node;
+import org.kitodo.dataaccess.NodeReference;
+import org.kitodo.dataaccess.ObjectType;
+import org.kitodo.dataaccess.RDF;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+/**
+ * Transforms a linked data structure to an XML file.
+ *
+ * @author Matthias Ronge
+ */
+public class XMLWriter {
+    /**
+     * String to pass to the transformer to switch on indenting.
+     */
+    private static final String TRANSFORMER_INDENT_TRUE = "yes";
+
+    /**
+     * String to pass to the transformer to set the indenting width in
+     * characters.
+     */
+    private static final String TRANSFORMER_INDENT_VALUE = "{http://xml.apache.org/xslt}indent-amount";
```

Comment:
Is a closing `}` missing or the beginning `{` wrong ?

Reply:
No. Apache just wants *that* string.

Reply:
I oversee the closing `}` and that the Apache library need the string in that way is not understandable for me but if this library needs this that it should go in.

Reply:
This is based on different understanding of name spaces. The syntax is `{<name space>}<simple name>`.